generators/generator-base.js

```diff
@@ -1517,6 +1517,40 @@ module.exports = class extends PrivateBase {
         this.copyTemplate(source, dest, 'stripHtml', generator, opt, template);
     }
 
+    /**
+     * Copy Ts templates after stripping translation keys when translation is disabled.
+     *
+     * @param {string} source - path of the source file to copy from
+     * @param {string} dest - path of the destination file to copy to
+     * @param {object} generator - context that can be used as the generator instance or data to process template
+     * @param {object} opt - options that can be passed to template method
+     * @param {boolean} template - flag to use template method instead of copy
+     */
+    processTs(source, dest, generator, opt, template) {
```

Comment:
Is this method different from processJs? looks like you are relying on this to run prettier, I would rather just run prettier using the npm command from the generated project during the yarn install/build phase and avoid doing it here. If it needs to be done during generation then I would prefer to use "Transform output files through streams" section in http://yeoman.io/authoring/file-system.html